docs/reference/commandline/build.md

```diff
@@ -48,8 +48,11 @@ Options:
                                 '<network-name>|<network-id>': connect to a user-defined network
       --no-cache                Do not use cache when building the image
       --pull                    Always attempt to pull a newer version of the image
+      --progress                Set type of progress output (only if BuildKit enabled) (auto, plain, tty). 
```

Comment:
`(only takes effect if BuildKit is enabled)`

Reply:
should probably say what the default value is

Reply:
The only thing is that's according to the cli `--help` manual 
https://github.com/docker/cli/blob/master/cli/command/image/build.go#L157
I think it's better to keep them consistent.